### PR TITLE
Add more tweet's

### PR DESCRIPTION
**Pull request by 9999years**

### CODE REVIEW

File: add-tweet.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+function warn  { _log "$(BRYELLOW)"     "[warn] " "$@"; }
+function error { _log "$(BRRED)"        "[error]" "$@"; }
+function fatal { _log "$(BOLD)$(BRRED)" "[FATAL]" "$@"; exit 1; }
+function cmd   { _log "$(CYAN)"         "[run]  " "\$ $(BOLD)$(UNDERLINED)$*"; }
+# }}}
+
+
+info "Paste a tweet and then hit Ctrl-D:"
+tweet=$(</dev/stdin)
+if jq --arg tweet "$tweet" --exit-status '.[] | any(. == $tweet)' \
```
Comment (eiais) [inline add-tweet.sh:79]: doesn't work on my machine

**Comment by 9999years** [comment]
wtf why

On Tue, Sep 1, 2020, at 10:01 PM, Kathryn Spiers wrote:
> 

> *@eiais* commented on this pull request.

> 
> In add-tweet.sh <https://github.com/eiais/puppy.rs/pull/2#discussion_r481547770>:

> > +    echo -n "$color$level $PROG_NAME ${color}[$(now)]:" "$@"
+    RESET
+}
+
+function dbg   { _log "$(GRAY)"         "[debug]" "$@"; }
+function info  { _log "$(BRGREEN)"      "[info] " "$@"; }
+function warn  { _log "$(BRYELLOW)"     "[warn] " "$@"; }
+function error { _log "$(BRRED)"        "[error]" "$@"; }
+function fatal { _log "$(BOLD)$(BRRED)" "[FATAL]" "$@"; exit 1; }
+function cmd   { _log "$(CYAN)"         "[run]  " "\$ $(BOLD)$(UNDERLINED)$*"; }
+# }}}
+
+
+info "Paste a tweet and then hit Ctrl-D:"
+tweet=$(</dev/stdin)
+if jq --arg tweet "$tweet" --exit-status '.[] | any(. == $tweet)' \

File: add-tweet.sh (hunk 1 above)
Comment (9999years) [reply add-tweet.sh:79]: install jq bitch

**Comment by eiais** [comment]
Love you

**Pull request merged**